platformio.ini

```diff
@@ -2,7 +2,7 @@
 ; https://docs.platformio.org/page/projectconf.html
 
 [platformio]
-default_envs = tbeam
```

Comment:
you probably didn't mean to check in this file :)

Reply:
For sure I need to check in this file. It adds the new RAK4631 variant. Just forgot to change back the default platform.

Just submitted correct platformio.ini file.